agent/agent_test.go

```diff
@@ -1008,11 +1008,46 @@ func TestAgent_Metadata(t *testing.T) {
 	})
 
 	t.Run("Many", func(t *testing.T) {
+		t.Parallel()
+		script := "echo -n hello"
+		if runtime.GOOS == "windows" {
+			script = "powershell " + script
```

Comment:
This doesn't seem to be working, CI failing with missing hello. Does it need quoting?